src/pages/Register.vue

```diff
@@ -191,7 +191,7 @@ export default {
         this.error = e
       } finally {
         this.loading = false
-        this.$router.push('/')
+        this.$router.go(-2) // Assumes that user came via /login page
```

Comment:
Is that a requirement?
Because honestly, I don't like this.

Reply:
As far as I understood @n1ghty1993, yes. That's how I understand the todo: "Weiterleitung nach Registrierung und Login"